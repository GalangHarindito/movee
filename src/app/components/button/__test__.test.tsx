import "@testing-library/jest-dom";
import { render, screen, fireEvent } from "@testing-library/react";
import Button from "./button";

describe('Button Test Component', () => {
    const onClick = jest.fn()
    it('renders button layout unchanged ', () => {

        const props = {
            label: 'Submit',
            onClick: onClick,
            buttonStyle: 'primary',
        }
        const { container } = render(
            <Button {...props} />
        );
        expect(container).toMatchSnapshot();
    })
})