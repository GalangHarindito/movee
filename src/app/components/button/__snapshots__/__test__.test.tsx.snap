// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Test Component renders button layout unchanged  1`] = `
<div>
  <div
    class="sc-gsTDqH dtnoZ"
  >
    <button
      class="sc-bdfCDU jdFpGB primary"
    >
      Submit
    </button>
  </div>
</div>
`;
